README.md

```diff
@@ -20,6 +20,15 @@ configure_args = "-Dusethreads"
 
 ```
 
+Alternatively, you can define 'app' as 'app_github' instead, so the destination 'app' directory will be a git repo clone instead of a directory copy.
+'app_dir_name' is the name of the cloned repo directory under '<output>/app'.
+
+```
+app_github = "https://github.com/gh_user/CoffeeChair.git"
+app_dir_name = "CoffeeChair"
```

Comment:
no, at not in a way that I could figure out yet. if you provide git clone with a destination directory it will create it as the directory of the clone itself, not create it under it... unlike doing it under current dir. a possible work-around is to do: `pushd $target; git clone $url; popd`.
then, we need to know the name for further use. if we let the clone to set that name, we'll need to list the created directory from the target one for further use. possible, just slightly more complicated than having the user set the name in the config and avoid the trouble (which i think is a reasonable first step, which can be improved later).